numpy/ma/core.py

```diff
@@ -2755,6 +2755,62 @@ class MaskedArray(ndarray):
         in any order (either C-, Fortran-contiguous, or even discontiguous),
         unless a copy is required, in which case it will be C-contiguous.
 
+     Examples
+    --------
+
+    You can initialize MaskedArray with ``mask`` attribute being array
+    of boolean values with the same shape as ``data`` attribute:
+    >>> np.ma.MaskedArray(np.ones([1, 2]), mask=[[False, False]])
```

Comment:
Using the same input for every example (i.e. `np.ones([1, 2])`) is definitely an improvement.

In my opinion, the examples could be improved further by using a slightly more complex input array to help highlight the features. For example, instead of using `np.ones([1, 2])` as `data`, how about `np.arange(6).reshape((2, 3))`? It's a slightly larger array, which better highlights the convenience of the scalar Boolean inputs to `mask` and having different numerical values for each element makes the examples more digestible.